src/signals/incident/components/form/ContainerSelect/WfsLayer/WfsLayer.tsx

```diff
@@ -0,0 +1,77 @@
+import type { FunctionComponent } from 'react';
+import { useMapInstance } from '@amsterdam/react-maps';
+import type { FeatureCollection } from 'geojson';
+import React, { useContext, useEffect, useState } from 'react';
+import { MAX_ZOOM_LEVEL, MIN_ZOOM_LEVEL } from '@amsterdam/arm-core/lib/constants';
+import type { DataLayerProps, WfsLayerProps } from '../types';
+
+import { fetchWithAbort } from '@amsterdam/arm-core';
+import type { ZoomLevel } from '@amsterdam/arm-core/lib/types';
+import ContainerSelectContext from '../context';
+import { NO_DATA, WfsDataProvider } from './context';
+
+export const isLayerVisible = (zoom: number, zoomLevel: ZoomLevel) => {
+  const { min, max } = zoomLevel;
+  return zoom <= (min ?? MIN_ZOOM_LEVEL) && zoom >= (max ?? MAX_ZOOM_LEVEL);
+};
+
+export interface Props extends WfsLayerProps {
+  children: React.ReactElement<DataLayerProps>;
```

Comment:
I believe there's no need to define the `children` prop explicitly.

Reply:
This is needed in order to call React.clone without type errors. Besides I want to ensure a child component is compulsory and it gets the expected props